libs/types/src/ui_theme.ts

```diff
@@ -15,7 +15,7 @@ export enum Color {
   DANGER_LOW_CONTRAST = '#ff3d3d',
   DANGER_MEDIUM_CONTRAST = '#820b0b',
   GRAY_DARK = '#222222',
-  GRAY_LIGHT = '#757575',
+  GRAY_LIGHT = '#8a8a8a',
```

Comment:
Missed this in the last PR, but saw an accessibility alert in storybook about this not passing the 4.5:1 contrast ratio requirement, so bumping up the brightness a bit (used as foreground in low-contrast theme)

Reply:
storybook gives you contrast ratio warnings? If so, very neat.

Reply:
Yeah! We currently have to remember to check the accessibility tab they provide, but I remember reading about some teams rolling storybook into their automated tests, so that they get that reporting on each commit.

That could also be a useful way for us to cut down on the duplication from writing tests, if the stories are just run as tests - hoping to look into that a little more after this project 🤞